src/gc/softwarewritewatch.h

```diff
@@ -11,6 +11,11 @@
 #ifdef FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP
 #ifndef DACCESS_COMPILE
 
+#ifndef GC_PAGE_SIZE
+#define GC_PAGE_SIZE 0x1000
+#endif
```

Comment:
this should not be GC_PAGE_SIZE - it has no relationship with GC_PAGE_SIZE. it should be its own thing. you can name it WRITE_WATCH_UNIT_SIZE.